docs/usage/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Renovate
+
+Learn how to troubleshoot problems with Renovate, where to find the logging output, and how to get help if needed.
+
+## Getting the logs
+
+Renovate logs the most recent jobs.
+Where you can find the logs depends on how you're running Renovate.
+
+### GitHub Hosted app
+
+Each pull request from Renovate contains a link to the WhiteSource Renovate Dashboard in the PR body text.
+
+The text you're looking for is:
+
+> This PR has been generated by WhiteSource Renovate. View repository job log here.
+
+Click on the blue text "here" to go to the WhiteSource Renovate Dashboard.
+Sign in with your GitHub or GitLab account.
+Once you're logged in, you can see the logs for the Renovate jobs on your repository.
+
+Renovate only keeps the logs for the last 3 days.
+
+When you've clicked on a recent job, you'll be able to select a debug level that you care about.
+For a full overview, make sure you select the DEBUG log level.
+
+### Selfhosted
+
+TODO: Explain log directory structure. Will probably be different depending on self-hosted platform...
+
+TODO: Explain how long logs are kept by default on differnet platforms.
+
+TODO: Explain how to toggle log level: <https://docs.renovatebot.com/examples/self-hosting/#about-the-log-level-numbers>
+
+## Log debug levels
+
+There are different severity levels for the log output.
+From least severe to most severe:
+
+- DEBUG
+- INFO
+- WARN
+- ERROR
+- FATAL
+
+As you can probably imagine, the stuff that's really going wrong will likely have a level of WARN or higher.
+
+TODO: What do the Renovate maintainers recommend as debug level?
+
+## Relevant part of the logs
+
+Please locate the relevant parts of the logs before asking for help or posting a bug report.
+Do not expect the Renovate maintainers to read through the full logs when trying to help you, as that takes a lot of time on our part.
+
+We recommend you follow this process:
+
+1. Read the _whole_ log output, to get a feel for all the steps that Renovate is taking
+1. Select - or search for - a higher debug level, like WARN, ERROR or FATAL
+1. Comparing the log output from a "bad" job to the log output from a known good job
+1. Copy/paste the relevant parts of the logs into your discussion post or bug report
+
+## Getting help
+
+If you cannot fix the problem yourself after reading the logs, and reading - or searching through - our documentation, search the [renovatebot/renovate discussion](https://github.com/renovatebot/renovate/discussions) forum to see if somebody has asked a similar or related question.
```

Comment:
The goal of this section is to try to nudge people into helping themselves, before they're going to spend maintainer time to get their problem fixed.